dev/PowerNotifications/ClosedSourceMock.cpp

```diff
@@ -0,0 +1,271 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
```

Comment:
It's worth putting a comment here calling this out as a *temporary* test strategy, which is only in place until the actual cross-binary ABI is available for mocking. If this file ever ends up in the main branch, something has gone wrong...